### PR TITLE
Revamp/scripts

### PR DESCRIPTION
**Pull request by mredolatti**

### CODE REVIEW

File: docker/functions.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/bin/env bash
```
Comment (Israphel) [inline docker/functions.sh:1]: should be /usr/bin/env under normal circumstances

also, alpine won't find bash.

File: splitio/commitversion.go [hunk 2]
```diff
@@ -5,4 +5,4 @@ This file is created automatically, please do not edit
 */
 
 // CommitVersion is the version of the last commit previous to release
-const CommitVersion = "a83d5f7"
\ No newline at end of file
+const CommitVersion = "409f9f0"
```
Comment (Israphel) [inline splitio/commitversion.go:8]: do you know someone called end of line?

File: docker/functions.sh (hunk 1 above)
Comment (mredolatti) [reply docker/functions.sh:1]: fixed. thank you fadon!

File: splitio/commitversion.go (hunk 2 above)
Comment (mredolatti) [reply splitio/commitversion.go]: go away

**Review by Israphel — APPROVED** [review]

**Pull request merged**